src/Process-Data/Move_Clean_Odds_Data_To_DB.py

```diff
@@ -0,0 +1,22 @@
+import os
+import sqlite3
+import pandas as pd
+import sys
+from tqdm import tqdm
+sys.path.insert(1, os.path.join(sys.path[0], '..'))
+from Utils.Dictionaries import team_codes
+
+directory = os.fsdecode('../../Odds-Data/Odds-Data-Clean')
+con = sqlite3.connect("../../Data/odds.sqlite")
+
+for file in tqdm(os.listdir(directory)):
+    filename = os.fsdecode(file)
+
+    try:
+        df = pd.read_excel(f"../../Odds-Data/Odds-Data-Clean/{file}") # create DataFrame by reading Excel
+        df.to_sql(f"odds_{file[:-5]}", con, if_exists="replace")
+
+    except Exception as e:
+        print(e)
+
+con.close()
```

Comment:
Can we skip this and just go right from the get_data script to DB?

Reply:
Yea, I don't see any need for this script in the future.  I just used to move the data that was already scraped so I didn't have to re-gather all of it.